modules/Core/src/main/java/org/terasology/logic/actions/TunnelAction.java

```diff
@@ -50,15 +57,25 @@
     private EntityManager entityManager;
 
     private Random random = new FastRandom();
+    private List<Optional<StaticSound>> explosionSounds = Lists.newArrayList();
 
     @Override
     public void initialise() {
+        explosionSounds.add(Assets.getSound("core:explode1"));
+        explosionSounds.add(Assets.getSound("core:explode2"));
+        explosionSounds.add(Assets.getSound("core:explode3"));
+        explosionSounds.add(Assets.getSound("core:explode4"));
+        explosionSounds.add(Assets.getSound("core:explode5"));
```

Comment:
my personal prefered way of getting assets is via @In AssetManager assetManager  as it is not a static access and could thus be mocked by for example a test.